File: 151001/Danko/RESTApplication/src/main/java/com/example/restapplication/mappers/TagMapper.java
package com.example.restapplication.mappers;

import com.example.restapplication.dto.TagRequestTo;
import com.example.restapplication.dto.TagResponseTo;
import com.example.restapplication.entites.Tag;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring")
public interface TagMapper {
    Tag toTag(TagRequestTo tag);
    TagResponseTo toTagResponse(Tag tag);
}

File: 151003/Alekseeva/TaskSQL/TaskSQL/Mappers/TagMapper.cs
using Riok.Mapperly.Abstractions;
using TaskSQL.Dto.Request.CreateTo;
using TaskSQL.Dto.Request.UpdateTo;
using TaskSQL.Dto.Response;
using TaskSQL.Models;

namespace TaskSQL.Mappers;

[Mapper]
public static partial class TagMapper
{
    public static partial Tag Map(UpdateTagRequestTo updateTagRequestTo);
    public static partial Tag Map(CreateTagRequestTo createTagRequestTo);
    public static partial TagResponseTo Map(Tag tag);
    public static partial IEnumerable<TagResponseTo> Map(IEnumerable<Tag> tags);

    public static partial IEnumerable<Tag> Map(
        IEnumerable<UpdateTagRequestTo> tagRequestTos);
}